dashboard/app/models/census/census_summary.rb

```diff
@@ -26,6 +26,67 @@ class Census::CensusSummary < ApplicationRecord
     HISTORICAL_YES: "HY",
     HISTORICAL_NO: "HN",
     HISTORICAL_MAYBE: "HM",
+    EXCLUDED: "E",
+    UNKNOWN: "U",
   }.freeze
   enum teaches_cs: TEACHES
+
+  # Use the zero byte as the quote character to allow importing double quotes
+  #   via http://stackoverflow.com/questions/8073920/importing-csv-quoting-error-is-driving-me-nuts
+  CSV_IMPORT_OPTIONS = {col_sep: ",", headers: true, encoding: 'bom|utf-8'}.freeze
+
+  # Seeds all the data from the S3 file.
+  def self.seed_all
```

Comment:
this took 13 minutes to seed the first time on my local machine, which is too much time for us to add to how long it takes to set up a new environment. partially following the pattern used in school.rb, please skip seeding this data when `CDO.stub_school_data` is true. this flag is set everywhere but production (the only place we need this data), and the census data is very closely tied to school data, so this makes me think it is a good idea to reuse this flag rather than create a new one.

Reply:
Good idea! Will do!